data/mods/Magiclysm/items/ethereal_items.json

```diff
@@ -34,7 +34,7 @@
     "symbol": ",",
     "color": "yellow",
     "use_action": [ { "type": "firestarter", "moves": 20 } ],
-    "flags": [ "TRADER_AVOID", "FIRESTARTER", "FIRE", "LIGHT_8", "FLAMING", "WATER_EXTINGUISH" ]
+    "flags": [ "TRADER_AVOID", "FIRESTARTER", "FIRE", "LIGHT_8", "FLAMING", "WATER_EXTINGUISH", "NO_UNWIELD"  ]
```

Comment:
this is the spell description: `Summons a small flame that does not burn you, but you can use it to light things on fire.  It seems to need you to have some intent to light things on fire, because you are able to put it in your pocket with no issue.`

Reply:
I don't know why this was marked as resolved given there has been no subsequent commit related to it, but seeing as how korggent just made it clear your change is incorrect for the description and intent of the spell, you should probably be removing the change.

Reply:
Sorry, I am not sure what happened. For personal reasons, I don’t have a computer at the moment, I will close it.

Reply:
I thought I changed it back, but in fact it didn't, is that the case?